lighthouse-core/gather/driver.js

```diff
@@ -353,6 +353,10 @@ class Driver {
    * @return {Promise<*>}
    */
   async _evaluateInContext(expression, contextId) {
+    // Use a higher than default timeout if the user hasn't specified a specific timeout.
+    // Otherwise, use whatever was requested.
+    const timeout = this._nextProtocolTimeout === DEFAULT_PROTOCOL_TIMEOUT ?
+      60000 : this._nextProtocolTimeout;
```

Comment:
since you just made `driver-test.js` shiny and new, could you add a quick test for this? I could see us easily breaking it at some point and the issue being difficult to notice

Reply:
good call! 👍